static/speakers.json

```diff
@@ -0,0 +1 @@
+[{"src":"2018-09-27-September-meetup.md","speakers":[{"title":"Video from Bornhack","name":"Julian"},{"title":"How we use Glimmer.js","name":"Kenneth Larsen"},{"title":"Data Modelling with Types","name":"Andreas Møller"},{"title":"RailsGirls Summer of Code","name":"Neer & Anne"}]},{"src":"2018-10-05-october-meetup.md","speakers":[{"title":"How to Maps with OpenStreetMap","name":"John Wika Haakseth"},{"title":"Why I use Wordpress as a backend","name":"Marijn Pool"},{"title":"Delivering DR TV with React.js","name":"Kjartan"}]},{"src":"2018-11-05-november-meetup.md","speakers":[{"title":"Using the new React Hooks","name":"Younes Meliani"},{"title":"Morphism and reusability - How we built a new product in record time","name":"Atli Adalsteinsson"},{"title":"How we use Storybooks at Vivino","name":"Johnathan Sewell"}]},{"src":"2019-01-24-january-meetup.md","speakers":[{"title":"micro:bit - JavaScript meets microcontroller","name":"Kenneth Geisshirt"},{"title":"GraphQL and React Suspense","name":"Morten Barklund"},{"title":"Shape your workflows with custom eslint plugins","name":"Peter Munter"}]},{"src":"2019-02-21-february-meetup.md","speakers":[{"title":"Talk with Neha Sharma from JSLovers community in Delhi","name":"Neha Sharma"},{"title":"WebBluetooth, WebUSB, LitElement","name":"Lars Knudsen"},{"title":"Markdownlint","name":"Jonas BN"}]},{"src":"2019-04-25-april-meetup.md","speakers":[{"title":"How we used Ionic 4 to create a PWA","name":"Mario Padilla"},{"title":"How we are building a new wine community with GraphQL","name":"Mads Obel"},{"title":"Changing Layout in Apps Immediately with Server-Driven UI","name":"Morten Bo"}]},{"src":"2019-05-23-may-meetup.md","speakers":[{"title":"Building framework-agnostic UI components","name":"Andreas Mehlsen"},{"title":"How we maintain more than 250 Web Components","name":"Rune Mehlsen"},{"title":"How to support Web Components across browsers with 1 line of code","name":"Frederik Wessberg"}]}]
```

Comment:
This shouldn't be committed, it will be generated just as part of the deployment step 😄